turbo.json

```diff
@@ -0,0 +1,34 @@
+{
+    "$schema": "https://turbo.build/schema.json",
+    "ui": "tui",
+    "tasks": {
+        "build": {
+            "dependsOn": ["^build"],
+            "outputs": ["dist/**", ".next/**"]
+        },
+        "lint": {
+            "dependsOn": ["//#eslint", "//#typecheck", "//#stylelint"]
+        },
+        "//#eslint": {
+            "outputs": ["node_modules/.cache/eslint"]
+        },
+        "eslint": {
+            "outputs": ["node_modules/.cache/eslint"]
+        },
+        "//#stylelint": {
+            "outputs": ["node_modules/.cache/stylelint"]
+        },
+        "stylelint": {
+            "outputs": ["node_modules/.cache/stylelint"]
+        },
+        "//#typecheck": {
+            "outputs": []
+        },
+        "typecheck": {
+            "outputs": []
+        },
+        "test": {
+            "outputs": []
```

Comment:
Given your message in DM, my understanding is that you haven't worked on the test yet but you do, I would expect on output for tests as well.